docs/getting_started/requirements.md

```diff
@@ -6,119 +6,108 @@ The following server requirements cover both running the software on-premise and
-        - Memcached 1.5 or higher (See [Memcached adapter](../guide/persistence_cache.md##memcached) for comparison with Redis)
-    - Session: either own Redis instance with persistence turned on, or Database.
-    - Search: Solr 7, Solr 8, or Elasticsearch 7.7 (recommended over SQL-based Search engine, especially on cluster, as SQL does not provide the same feature set or performance as the other two).
-    - IO: NFS or S3
-    - HttpCache, using one of:
-        - [Varnish](http://varnish-cache.org/) 6.0LTS with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.rst)
-        - [Fastly](https://www.fastly.com/) using [the bundle provided with [[= product_name_exp =]]](../guide/http_cache.md#serving-varnish-through-fastly)
-
--   PHP extensions/modules
-    -   curl
-    -   ctype
-    -   dom (usually bundled with `xml` extension package)
-    -   fileinfo
-    -   iconv
-    -   intl
-    -   mbstring
-    -   json
-    -   opcache
-    -   pdo
-        - pdo mysql *(with mysqlnd)*
-        - pdo pgsql
-    -   posix
-    -   reflection
-    -   xml
-    -   xsl
-    -   zip
-    -   [php-redis](https://pecl.php.net/package/redis) *or* [php-memcached](https://pecl.php.net/package/memcached)
-
-
-### Development and Experimental setups
-
-[[= product_name =]] can theoretically run and execute on many more setups than the ones listed as recommended and supported, including any [operating system supported by PHP](https://wiki.php.net/platforms), on a PHP 7.3 version or higher that pass the [Symfony requirements](http://symfony.com/doc/5.0/reference/requirements.html), using cache solutions technically supported by [Symfony Cache component](https://symfony.com/doc/5.0/components/cache/cache_pools.html), using databases supported by [Doctrine DBAL](https://www.doctrine-project.org/projects/doctrine-dbal/en/2.9/reference/configuration.html#driver), and using a binary file storage solution supported by [FlySystem](https://github.com/thephpleague/flysystem#adapters).
+The minimal setup requires PHP,  MySQL/MariaDB, Apache/Nginx, Node.js and `yarn`.
+Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached, NFS/EFS/S3 and Solr/Elasticsearch in a [clustered setup](../guide/clustering.md).
 
-Examples of Development setups:
+Using the latest supported version or each product or component is always recommended.
 
--   OS: Windows, macOS X, Linux
--   Filesystem: NTFS, HFS+/APFS, ...
+## Operating system
 
-Examples of Experimental setups:
+- Debian 10.x "Buster"
+- Ubuntu 20.04 "Focal Fossa"
+- RHEL / CentOS 8.1+
+
+## Web server
+
+- Nginx 1.12, 1.14, 1.16
+- Apache 2.4 (with required modules `mod_rewrite`, `mod_env` and recommended: `mod_setenvif`, `mod_expires`;
+event MPM is recommended, if you need to use prefork you also need the `mod_php` module)
+
+## DBMS
+
+- MariaDB 10.3, 10.4 (optionally 10.2 - deprecated)
+- MySQL 8.0 (optionally 5.7 - deprecated)
+- PostgreSQL 10+
+
+## PHP
+
+- 7.3
+- 7.4
```

Comment:
Should we add a "(recommended)" here for 7.4?

Reply:
I realise now that the "Using the latest listed version of each product or component is always recommended." makes this kind of redundant. But it doesn't hurt either, now that it's there. 🤷‍♂️